spec/support/sample.config.yml

```diff
@@ -8,7 +8,7 @@ mysql2:
   encoding: utf8
 
 postgres:
-  adapter: postgresql
+  adapter: postgis
```

Comment:
Perhaps it would be better to add a whole new group here? Maybe I'm missing something, but it doesn't seem right to be removing the vanilla (non-PostGIS) PostgreSQL adapter?

Reply:
@robotfelix I agree with you on this comment. 👍